src/app/common/KFilterBar.vue

```diff
@@ -443,6 +444,7 @@ function areFieldsSemanticallyIdentical(fieldsA: Fields, fieldB: Fields): boolea
   flex-grow: 1;
   width: 100%;
   border: none;
+  padding: 12px;
```

Comment:
**Question**: Why was this added?

Reply:
This should be handled where KFilter is used to avoid the component to _always_ have an arbitrary padding.

Reply:
I _always_ want it to have this padding, so I want to add it here.

Reply:
We only ever use this in a KTable toolbar, all things in the toolbar should have the same height so they all line up nicely. We also use KSelect in the toolbar. Giving KFilterBar this padding makes it the same height as the KSelect.

i.e. I _always_ want this to be this height as this is only ever used in the toolbar alongside a KSelect

Reply:
Again, my thoughts exactly. Is there anything else outstanding here?